healvis/cosmology.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import sys
+import numpy as np
+from astropy.cosmology import Planck15 as cosmo
+from astropy.constants import c
+
+# -----------------------
+#  Functions and constants related
+#  to conversions between instrument and
+#  cosmological units.
+# -----------------------
+
+f21 = 1.420405751e9
+c_ms = c.to('m/s').value
+pi = np.pi
+
+
+def jy2Tsr(f, bm=1.0, mK=False):
+    '''Return [K sr] / [Jy] vs. frequency (in Hz)
+        Arguments:
+            f = frequencies (Hz)
+            bm = Reference solid angle in steradians (Defaults to 1)
+            mK = Return in mK sr instead of K sr
+    '''
+    c_cmps = c_ms * 100.   # cm/s
+    k_boltz = 1.380658e-16   # erg/K
+    lam = c_cmps / f  # cm
+    fac = 1.0
+    if mK:
+        fac = 1e3
+    return 1e-23 * lam**2 / (2 * k_boltz * bm) * fac
+
+
+def dL_df(z):
+    '''[Mpc]/Hz, from Furlanetto et al. (2006)'''
```

Comment:
Is this comoving or physical Mpc? I think its comoving. If it is, let's be specific via `[cMpc / Hz]`